### PR TITLE
Added github link. Changed the link in rules. Adjusted about page to …

### PR DESCRIPTION
**Pull request by WritingPanda**
Added github link. Changed the link in rules. Adjusted about page to reflect changes. Added blog post.

### CODE REVIEW

File: content/articles/goodbye-2019-hello-2020.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Our community also started two open source projects: Barbacoa and PuroEvents. People within our community have the opportunity to dive in and contribute to those projects as they see fit, and that helps new developers get into open source software development, as well. It's a great way for experienced developers to mentor new people, too, so the opportunity is there for everyone.
```
Comment (gtmanfred) [inline content/articles/goodbye-2019-hello-2020.md]: I think this is disingenuous, we kind of started barbacoa, but never really did anything, and PuroEvents only has a readme and nothing else.

File: pelicanconf.py [hunk 2]
```diff
@@ -30,6 +30,7 @@
 LOGO = 'sadev_500w.png'
 MENUITEMS = [
     ('slack', 'https://join.slack.com/t/sanantoniodevs/shared_invite/MjE2ODI0NDQ4NjkwLTE1MDA5MDgwODctMzcxMmZhNjE0Zg'),
+    ('github', 'https://github.com/SADevs/sadevs.github.io'),
```
Comment (gtmanfred) [inline pelicanconf.py]: this should probably just be /SADevs and not the link to the website repository. but i don't have super strong feelsing about it

File: content/articles/goodbye-2019-hello-2020.md (hunk 1 above)
Comment (WritingPanda) [reply content/articles/goodbye-2019-hello-2020.md]: I disagree. It's a start of something. The thing that needs to be done is organizing people and getting them involved. If you write open source code and no one knows about it, is it really open source?

File: pelicanconf.py (hunk 2 above)
Comment (WritingPanda) [reply pelicanconf.py]: I was more thinking that if someone wanted to see the repo where the code was held, then they'd have direct access. I could do two links: Github org page and repo for the site.

**Pull request merged**